### DISCUSSION TITLE
I lost the setup code…

### DISCUSSION THREAD

**Discussion by Domegraton** [Pryaxis/TShock#3009] [Q&A]
So I was completely new to tshock and tried stuff out and found in the logs that I can use a /setup command. But later I discovered that I can’t move town NPCs with out admin. Watched a tutorial and need the setup code… but it’s not in the logs anymore and no one has admin. What can I do? I also can’t download any server files. (I’m using Hostfactor.io if that matters)

**Comment by ghost** [comment]
Uh oh...

**Comment by Mooshay105** [comment]
Do you have access to the console?
If so, you could run (In console):
`/user group <username> owner`
--or--
`/user group <username> superuser`

**Reply by ghost** [reply]
"`/user group <username> superuser`" is wrong, i believe it's supposed to be "`/user group <username> superadmin`"

**Reply by ghost** [reply]
Sorry to be so critical...

**Reply by Mooshay105** [reply]
Its fine, It has been a while since I have needed use that command.

**Comment by zyrafaq** [comment]
use user add group <username> superadmin!
